cfgov/v1/tests/test_author_names.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from v1.models.base import CFGOVPage
+
+class TestAuthorNames(TestCase):
+
+    def test_names_get_sorted_alphabetically(self):
```

Comment:
It is intentional that this tests `get_authors` and not `alphabetize_authors` as `get_authors` is the function called by the templates. If we ever decide to go with another sort order, this test will fail, which I think is fine.